db/db_local/backup.go

```diff
@@ -20,8 +20,10 @@ type dbStartConfig struct {
 	dbName string
 }
 
-// prepareBackup creates a backup file for the current database if it is not the same version
-// that is required
+// prepareBackup creates a backup file of the public schema for the current database
```

Comment:
// prepareBackup creates a backup file of the public schema for the current database, if we are migrating
// if a backup was taken, this returns the name of the database that was backed up